module/VuFind/tests/unit-tests/src/VuFindTest/Connection/LibGuidesTest.php

```diff
@@ -66,6 +66,40 @@ public function testGetAccounts()
         $this->assertEquals('Melvil', $dewey->first_name);
     }
 
+    /**
+     * GetAZ test provider.
+     *
+     * @return array[]
+     */
+    public static function getAzProvider(): array
+    {
+        return [
+            [ true, 4 ],
+            [ false, 5 ],
+            [ null, 4 ],
```

Comment:
I would recommend naming the cases for context:

```suggestion
            'exclude hidden (explicitly)' => [ true, 4 ],
            'do not exclude hidden' => [ false, 5 ],
            'exclude hidden (by default)' => [ null, 4 ],
```

Reply:
Ah, didn't realize (forgot) provider could do that, nice!